plugins/grpc/interceptors/newrelic.go

```diff
@@ -77,7 +79,15 @@ func (i NewrelicInterceptor) Handler() grpc.UnaryServerInterceptor {
 			st, _ := status.FromError(err)
 			txn.AddAttribute("GrpcStatusMessage", st.Message())
 			txn.AddAttribute("GrpcStatusCode", st.Code().String())
-			txn.NoticeError(nrpkgerrors.Wrap(err))
+			var ae *app_grpc.ApplicationError
+			if errors.As(err, &ae) {
+				if !ae.Expected() {
+					txn.NoticeError(nrpkgerrors.Wrap(err))
```

Comment:
maybe add some more attributes to Newrelic
https://github.com/newrelic/go-agent/blob/master/GUIDE.md#error-reporting
```
txn.NoticeError(newrelic.Error{
    Message: "my error message",
    Class:   "IdentifierForError",
    Attributes: map[string]interface{}{
        "important_number": 97232,
        "relevant_string":  "zap",
    },
})
```